File: Selnium script for testing/Script with Python/test_scenario_35.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_35():
    """
    Dummy test run for scenario 35
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_35")

    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_35()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario35.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario35 {
    public static void executeTest35() {
        /*
         * Dummy test run for scenario 35
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_35");

        try {
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_363");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            Thread.sleep(2000);
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_584");
            driver.findElement(By.id("user_name")).click();
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_775");
            driver.findElement(By.className("nav-item")).sendKeys("test_data_684");
            driver.findElement(By.id("user_name")).sendKeys("test_data_933");
            driver.findElement(By.id("user_name")).sendKeys("test_data_733");
            Thread.sleep(2000);
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest35();
    }
}
